docs/modules/ROOT/pages/managing-viridian-clusters.adoc

```diff
@@ -0,0 +1,96 @@
+== Managing Viridian Clusters Using the Hazelcast CLC
+
+:description: In this tutorial, you'll learn the basics of managing Viridian clusters using CLC. You'll see how to create, list, delete clusters and how to download their logs. Also, you'll learn how to perform stop/resume operations on Viridian clusters using CLC.
+
+{description}
+
+== Before You Begin
+
+CLC  requires an API key and API secret to perform Viridian operations.
+You can create them on link:https://viridian.hazelcast.com/settings/developer[Viridian Developer Settings Page].
+After creating the Viridian API key and secret, run the following command for CLC to remember them:
+
+[source, bash]
+----
+clc viridian login
+----
+
+You will be asked for the API key and secret. Press kbd:[Enter] to confirm the input.
+If the API key and secret are correct, you will get the following output:
+[source, bash]
+----
+Viridian token was fetched and saved.
+OK
+----
+
+If you get an error, make sure your Viridian API key and secret are correct and try again.
+
+== Creating a Cluster on Viridian
+
+To create a new cluster on Viridian, you can use the following command:
+
+[source, bash]
+----
+clc viridian create-cluster --name my-cluster --development
+Imported configuration: my-cluster
+OK
+----
+This command creates a development cluster named `my-cluster`.
+
+== Listing Clusters on Viridian
+
+To list existing clusters on Viridian, you can use the following command:
+
+[source, bash]
+----
+clc viridian list-clusters
+----
+This command lists the clusters, and you will see such an output:
+[source, bash]
+----
+x1qvpphn        my-cluster     RUNNING 5.3.0-BETA-2
+OK
+----
+
+== Downloading the Logs of the Viridian Cluster
```

Comment:
Could you add a section before this one that shows how to use the configuration imported after the cluster is created? It could be something simple, like `map set` and then `map get`.